### PR TITLE
npmify

### PR DESCRIPTION
**Pull request by mdarens**
- add linting
- update tests to mocha

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,25 @@
 {
-  "name": "knockout-init",
-  "version": "1.1.0",
+  "name": "knockout.init",
+  "version": "1.0.0",
```
Comment (chadly) [inline package.json]: why are you going backwards in versions?

File: package.json [hunk 2]
```diff
@@ -1,18 +1,25 @@
...
   "description": "This is a simple knockout component that will automatically create observables from arbitrary json",
-  "main": "app.js",
+  "main": "knockout.init.js",
   "scripts": {
-    "test": "grunt test"
+    "lint": "eslint . --format=unix",
+    "test": "mocha"
```
Comment (chadly) [inline package.json]: add a prepublish step that calls `npm run lint && npm test`

File: package.json (hunk 1 above)
Comment (mdarens) [inline package.json]: Trying to relive my childhood I suppose.

**Comment by chadly** [comment]
:shipit:

**Pull request merged**